src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -170,17 +184,23 @@ private async Task SynchronizeSuppressedIssuesAsync()
                 // TODO: Handle race conditions
                 var moduleKeyToRelativePathToRoot = (await this.sonarQubeService.GetAllModulesAsync(sonarQubeProjectKey,
                         cancellationTokenSource.Token))
-                    .ToDictionary(x => x.Key, x => x.RelativePathToRoot ?? string.Empty);
-                this.hasModules = moduleKeyToRelativePathToRoot.Keys.Count == 1;
+                    .ToDictionary(x => x.Key, x => x.RelativePathToRoot?.Trim('\\', '/').Replace('/', '\\') ?? string.Empty);
```

Comment:
The _Trim_ and _Replace_ is repeated on line 226 below. It would be better to have a _NormalizePath_ function that handled the null checking and transformation.